helpers/file_helpers.rb

```diff
@@ -4,4 +4,12 @@ def self.out_path(in_path)
     filename = File.basename(in_path, ".*")
     return "#{path}/#{filename}.mp4"
   end
+
+  def self.concat_out_path(first_in, second_in)
+    path = File.dirname(first_in)
+    first_filename = File.basename(first_in, ".*")
+    second_filename = File.basename(second_in, ".*")
+
+    return "#{path}/#{first_filename + second_filename}.mp4"
```

Comment:
For this filename thing, this is pretty flexible, but I was thinking that we'll probably need a naming convention, particularly when we get to the point of uploading to S3. In that case, we could make assumptions about the names so that we get something more consistent.

For instance, if multipart files are named `test-pt-1.mov` and `test-pt-2.mov` or something like that, then the output would ideally be `test.mov`. Granted that's a bit more work, but without enforcing a file naming scheme, I foresee some complications.

But perhaps this is a moot point since the joined file may be most an intermediate file anyways. Or also moot in the case where we provide an option for the user of the script to provide an output filename.

Thoughts?

Reply:
I think an output filename is easily the best solution, there's no ambiguity, I somehow didn't think of that myself.